benchmark/nixlbench/contrib/Dockerfile.aws-amzn2023

```
@@ -0,0 +1,260 @@
+ARG HWLOC_VERSION="2.12.2"
+ARG EFA_VERSION="1.48.0"
+ARG UCX_REF="v1.20.x"
+ARG ARCH="x86_64"
+ARG WHL_PYTHON_VERSIONS="3.12"
+ARG WHL_PLATFORM="manylinux_2_39_$ARCH"
+ARG BUILD_TYPE="release"
+ARG NPROC
+ARG ABSL_TAG="lts_2025_08_14"
+ARG GRPC_TAG="v1.73.0"
+
+RUN dnf install -y \
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)